src/simulation/m_riemann_solvers.fpp

```diff
@@ -1512,12 +1531,14 @@ contains
                     end do
 
                 elseif (model_eqns == 2 .and. bubbles) then
```

Comment:
does this feature work with `bubbles == .TRUE.`? If not, then remove the modified code. IF it does, then there should be CI tests for it